src/pages/Transfer/CrossChainTransferForm/index.tsx

```diff
@@ -115,11 +117,9 @@ const CrossChainTransferForm = (): JSX.Element => {
     }
   };
 
-  const handleUpdateUsdAmount = (event: any) => {
-    if (!event.target.value) return;
```

Comment:
are you confident we don't need this early return?

Reply:
Good eye. I didn't meant to remove this. Thank you.